puppetboard/app.py

```diff
@@ -708,15 +708,17 @@ def fact(env, fact, value):
             value_json = value_object
         else:
             value_json = dumps(value_object)
-
+    natural_time_delta_sort = True
     return render_template(
```

Comment:
This should be set to True only if the name of the fact is one of „uptime”, etc.